### ISSUE TITLE
dbc.AccordionItem does not handle components as children properly

### ISSUE THREAD

**Issue by pierrebchn** [facultyai/dash-bootstrap-components#717]
- dash version: `2.0.0`
- dash-bootstrap-components version: `1.0.0b3`
- components affected by bug: AccordionItem

### What is happening?

The component AccordionItem only takes as children a string or a list of strings. The other components are not managed.

### What should be happening?

It should accept any component like dbc.Container or a list of components. Here is below an example where it works with a string as children and where it doesn't with a html.P as children.

### Code

```python
import dash
import dash_bootstrap_components as dbc
from dash import Input, Output, dcc, html

app = dash.Dash(external_stylesheets=[dbc.themes.BOOTSTRAP])

# the style arguments for the sidebar. We use position:fixed and a fixed width
SIDEBAR_STYLE = {
    "position": "fixed",
    "top": 0,
    "left": 0,
    "bottom": 0,
    "width": "16rem",
    "padding": "2rem 1rem",
    "background-color": "#f8f9fa",
}

# the styles for the main content position it to the right of the sidebar and
# add some padding.
CONTENT_STYLE = {
    "margin-left": "18rem",
    "margin-right": "2rem",
    "padding": "2rem 1rem",
}

sidebar = html.Div(
    [
        html.H2("Sidebar", className="display-4"),
        html.Hr(),
        dbc.Nav(
            [
                dbc.NavLink("Working", href="/", active="exact"),
                dbc.NavLink("Not Working", href="/page-1", active="exact"),
            ],
            vertical=True,
            pills=True,
        ),
    ],
    style=SIDEBAR_STYLE,
)

content = html.Div(id="page-content", style=CONTENT_STYLE)

app.layout = html.Div([dcc.Location(id="url"), sidebar, content])


@app.callback(Output("page-content", "children"), Input("url", "pathname"))
def render_page_content(pathname):

    if pathname == "/":
        return html.Div(
            dbc.Accordion(
                [
                    dbc.AccordionItem(
                        "This is the content of the first section", title="Item 1"
                    ),
                    dbc.AccordionItem(
                        "This is the content of the second section", title="Item 2"
                    ),
                ],
                flush=True,
            ),
        )

    elif pathname == "/page-1":
        return html.Div(
            dbc.Accordion(
                [
                    dbc.AccordionItem(
                        html.P("This is the content of the first section"),
                        title="Item 1",
                    ),
                    dbc.AccordionItem(
                        "This is the content of the second section", title="Item 2"
                    ),
                ],
                flush=True,
            ),
        )


if __name__ == "__main__":
    app.run_server(port=8888)

**Comment by tcbegley** [comment]
Thanks for raising this @pierrebchn and the helpful example.

Can confirm I see the same issue. Will look into a fix.

**Comment by tcbegley** [comment]
This is resolved in `1.0.0rc1`. See example in docs [here](https://dbc-v1.herokuapp.com/docs/components/accordion/)

Update with

```
pip install -U --pre dash-bootstrap-components
```
